src/crkeng/site/settings.py

```diff
@@ -25,15 +25,16 @@
 
 INSTALLED_APPS.insert(0, "crkeng.app")
 
-ROOT_URLCONF = "crkeng.site.urls"
-
 WSGI_APPLICATION = "crkeng.site.wsgi.application"
 
 # Morphodict configuration
 
 # The ISO 639-1 code is used in the lang="" attributes in HTML.
 MORPHODICT_ISO_639_1_CODE = "cr"
 
+MORPHODICT_SOURCE_LANGUAGE = "crk"
+MORPHODICT_TARGET_LANGUAGE = "eng"
+
 # What orthographies -- writing systems -- are available
```

Comment:
Pedantic point, totally just an FYI: Writing systems and orthographies are slightly different. A writing system is a group of associated symbols, like the Latin alphabet. An orthography is how a particular writing system is _used_ to write a particular language. So the Spanish and English _orthographies_ both use the Latin _writing system_.

The ISO codes are for writing systems, and the subcodes allow you to specify variants of the writing system. For example, the variant of the Latin writing system that has acute accents on vowels is used for the Spanish orthography, while the variant of the Latin writing system without accents on vowels is used for the English orthography.